melodies_monet/plots/surfplots.py

```diff
@@ -769,3 +906,145 @@ def make_boxplot(comb_bx, label_bx, ylabel = None, vmin = None, vmax = None, out
     
     plt.tight_layout()
     savefig(outname + '.png', loc=4, logo_height=100)
+
+def make_spatial_exceedance(df, column_o=None, label_o=None, column_m=None,
+                      label_m=None, ylabel = None, vdiff=None,
+                      outname = 'plot',
+                      domain_type=None, domain_name=None, fig_dict=None,
+                      text_dict=None,debug=False):
+
+    """Creates surface spatial bias plot. 
+    
+    Parameters
+    ----------
+    df : dataframe
+        model/obs pair data to plot
+    column_o : str
+        Column label of observation variable to plot
+    label_o : str
+        Name of observation variable to use in plot title 
+    column_m : str
+        Column label of model variable to plot
+    label_m : str
+        Name of model variable to use in plot title
+    ylabel : str
+        Title of colorbar axis
+    vdiff : real number
+        Min and max value to use on colorbar axis
+    outname : str
+        file location and name of plot (do not include .png)
+    domain_type : str
+        Domain type specified in input yaml file
+    domain_name : str
+        Domain name specified in input yaml file
+    fig_dict : dictionary
+        Dictionary containing information about figure
+    text_dict : dictionary
+        Dictionary containing information about text
+    debug : boolean
+        Whether to plot interactively (True) or not (False). Flag for 
+        submitting jobs to supercomputer turn off interactive mode.
+        
+    Returns
+    -------
+    plot 
+        surface bias plot
```

Comment:
This refers to the return value of the function, i.e. if you do `a = make_spatial_exceedance(...)`, what is `a`? It doesn't look you have specified to return anything, so this function just returns `None`, in which case you can document that or just leave off the "Returns" section of the docstring.